build/gulpfile.vscode.js

```diff
@@ -140,8 +140,8 @@ const bundleVSCodeTask = task.define('bundle-vscode', task.series(
 				resources: vscodeResources,
 				fileContentMapper: filePath => {
 					if (
-						filePath.endsWith('vs/code/electron-sandbox/workbench/workbench.js') ||
-						filePath.endsWith('vs/code/electron-sandbox/processExplorer/processExplorer.js')) {
+						filePath.endsWith(path.join('vs', 'code', 'electron-sandbox', 'workbench', 'workbench.js')) ||
```

Comment:
I think I would prefer us to convert backslash to slash and compare with the path as it was to ensure we are not running into other issues.